integration-tests/modules/__tests__/link-modules/define-link.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { medusaIntegrationTestRunner } from "medusa-test-utils"
+import CurrencyModule from "@medusajs/currency"
+import RegionModule from "@medusajs/region"
+import { defineLink } from "@medusajs/utils"
+import { MedusaModule } from "@medusajs/modules-sdk"
+
+jest.setTimeout(50000)
+
+medusaIntegrationTestRunner({
+  testSuite: ({ getContainer }) => {
+    describe("defineLink", () => {
+      let appContainer
+      let remoteQuery
+
+      beforeAll(async () => {
+        appContainer = getContainer()
+        remoteQuery = appContainer.resolve("remoteQuery")
+      })
+
+      it("should generate a proper link definition", async () => {
+        const currencyLinks = CurrencyModule.links
+        const regionLinks = RegionModule.links as any // TODO fix typings for model.define with config object
```

Comment:
comment: Separately, @shahednasser highlighted that `links` could be understood as existing links instead of "something that can be linked", I think that was a good point. Maybe we should also consider renaming this to `linkConfig` or `linkable` or something third.

What do you think?

Reply:
linker?